extensions/amp-lightbox-gallery/0.1/service/lightbox-manager-impl.js

```diff
@@ -18,7 +18,6 @@ import {AmpEvents} from '../../../../src/amp-events';
 import {CommonSignals} from '../../../../src/common-signals';
 import {
   LIGHTBOX_THUMBNAIL_AD,
-  LIGHTBOX_THUMBNAIL_GIF,
```

Comment:
Is this value still used anywhere in the code? Can it be removed from `lightbox-placeholders.js`?

Reply:
I was debating this, but I think yes.